### PR TITLE
POLIO-1622 Chronogram: Have a way to enter activities in FR & EN

### PR DESCRIPTION
**Pull request by kemar**
Chronogram: Have a way to enter activities in FR & EN

Related JIRA tickets : [POLIO-1622](https://bluesquare.atlassian.net/browse/POLIO-1622)

## Changes

- use a `TranslatedField` for `description` fields in `ChronogramTemplateTask` and `ChronogramTask`
- install `gettext` and run `makemessages` in the CI workflow (this allows to test translations)
- allow to enter activities in FR and EN in the UI of "chronogram details" and "chronogram template"

## How to test

1. **go to `http://localhost:8081/dashboard/polio/chronogram/templateTask`**
    -add or edit a template
    -you should have 2 inputs to enter activities in EN and FR
    -only EN is mandatory and will be the default value displayed if no FR value exists
2. **go to a chronogram detail page**
    -  e.g.`http://localhost:8081/dashboard/polio/chronogram/accountId/1/chronogram_id/7`
    -add or edit a task
    -you should have 2 inputs to enter activities in EN and FR
    -only EN is mandatory and will be the default value displayed if no FR value exists
3. **go to `localhost:8081/dashboard/settings/users`** (or use the button at the bottom of the nav menu)
    -change your UI language
4. **go back to either URL of step `1` or step `2`**
    -the translation for your active language should be displayed in tables

## Video

https://github.com/user-attachments/assets/7da6e919-67a6-46b3-9559-1d42f15c9189

[POLIO-1622]: https://bluesquare.atlassian.net/browse/POLIO-1622?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -107,8 +107,8 @@ jobs:
         uses: actions/setup-python@v2
         with:
           python-version: 3.9
-      - name: Install GDAL
-        run: sudo apt-get update && sudo apt install gdal-bin 
+      - name: Install GDAL and gettext
+        run: sudo apt-get update && sudo apt install gdal-bin && sudo apt install gettext
```
Comment (bmonjoie) [inline .github/workflows/main.yml]: Nit-picking, but why not:
```suggestion
        run: sudo apt-get update && sudo apt install gdal-bin gettext
```

File: plugins/polio/js/src/domains/Chronogram/ChronogramDetails/Modals/ChronogramTaskCreateEditModal.tsx [hunk 2]
```diff
@@ -112,11 +113,18 @@ const CreateEditChronogramTaskModal: FunctionComponent<Props> = ({
...
+                        required
+                    />
+                </Box>
+                <Box mb={2}>
+                    <Field
+                        label={formatMessage(MESSAGES.labelDescriptionFr)}
+                        name="description_fr"
                         component={TextInput}
                         required
-                        disabled={!userHasReadAndWritePerm}
```
Comment (bmonjoie) [inline plugins/polio/js/src/domains/Chronogram/ChronogramDetails/Modals/ChronogramTaskCreateEditModal.tsx:119]: Is this not needed anymore?

File: plugins/polio/js/src/domains/Chronogram/ChronogramDetails/hooks/validation.ts [hunk 3]
```diff
@@ -11,10 +11,11 @@ export const useChronogramTaskSchema = () => {
             .string()
             .trim()
             .required(formatMessage(MESSAGES.validationFieldRequired)),
-        description: yup
+        description_en: yup
             .string()
             .trim()
             .required(formatMessage(MESSAGES.validationFieldRequired)),
+        description_fr: yup.string().trim(),
```
Comment (bmonjoie) [inline plugins/polio/js/src/domains/Chronogram/ChronogramDetails/hooks/validation.ts:18]: In the UI, there is an asterisk indicating that this field is required, but it is not here.

File: plugins/polio/js/src/domains/Chronogram/ChronogramTemplateTask/hooks/validation.ts [hunk 4]
```diff
@@ -11,10 +11,11 @@ export const useChronogramTemplateTaskSchema = () => {
             .string()
             .trim()
             .required(formatMessage(MESSAGES.validationFieldRequired)),
-        description: yup
+        description_en: yup
             .string()
             .trim()
             .required(formatMessage(MESSAGES.validationFieldRequired)),
+        description_fr: yup.string().trim(),
```
Comment (bmonjoie) [inline plugins/polio/js/src/domains/Chronogram/ChronogramTemplateTask/hooks/validation.ts:18]: In the UI, there is an asterisk indicating that this field is required, but it is not here.

File: plugins/polio/js/src/domains/Chronogram/ChronogramDetails/hooks/validation.ts (hunk 3 above)
Comment (kemar) [reply plugins/polio/js/src/domains/Chronogram/ChronogramDetails/hooks/validation.ts:18]: Fixed, well spotted, thanx!

File: plugins/polio/js/src/domains/Chronogram/ChronogramDetails/Modals/ChronogramTaskCreateEditModal.tsx (hunk 2 above)
Comment (kemar) [reply plugins/polio/js/src/domains/Chronogram/ChronogramDetails/Modals/ChronogramTaskCreateEditModal.tsx:119]: Yes it is. Fixed, thanx!

**Review by bmonjoie — APPROVED** [review]

**Pull request merged**